config/prow/plugins.yaml

```
@@ -157,6 +157,14 @@ label:
           - pushkarj
         # This label is used to filter/tag CVEs announced by SRC
         label: official-cve-feed
+    kubernetes/enhancements:
+      # Restrict setting of tracked/no and tracked/yes labels to the enhancements team.
+      - allowed_teams:
+        - enhancements
+        label: tracked/no
+      - allowed_teams:
+        - enhancements
+        label: tracked/yes
```

We should also add the `tracked/out-of-tree` label.

```suggestion
        label: tracked/yes
      - allowed_teams:
          - enhancements
          label: tracked/out-of-tree
```

---

The text for `tracked/out-of-tree` says that this label is used when items do not need to be tracked by the release team

Are we sure we want to restrict this one too?
`tracked/no`: Denotes an enhancement issue is NOT actively being tracked by the Release Team
`tracked/yes`: Denotes an enhancement issue is actively being tracked by the Release Team
`tracked/out-of-tree`: Denotes an out-of-tree enhancement issue, which does not need to be tracked by the Release Team

---

Probably yes. In the past people have incorrectly used that label and it caused issues. It makes sense for one group to own all permutations of tracked/* to ensure they are all correct.

---

Thanks for the context!